app/controllers/sessions.rb

```diff
@@ -0,0 +1,21 @@
+get '/sessions/new' do
+  erb :'sessions/new'
+end
+
+post '/sessions' do
+  puts params
+  @user = User.authenticate(params[:username], params[:password])
+  if @user
+    session[:user_id] = @user.id
+    "login success"
```

Comment:
FYI this line doesn't have any effect, be sure to remove any obsolete code/comments